### PR TITLE
Change XCTAssertTrue with XCTAssertEqual

### PR DESCRIPTION
**Pull request by srabuini**
Working on this exercise, I had a difference in one decimal. And I realized that would be better to replace all those `XCTAssertTrue` with `XCTAssertEqual` in order to get a better feedback.

Please, have a look and let me know.

Thanks!

### CODE REVIEW

File: exercises/space-age/SpaceAgeTest.swift [hunk 1]
```diff
@@ -1,60 +1,56 @@
...
 import XCTest
 
-
-
 class SpaceAgeTest: XCTestCase {
-    
     func testAgeInSeconds() {
         let age = SpaceAge(1_000_000)
-        XCTAssertTrue(1_000_000 == age.seconds)
+        print(age.seconds)
```
Comment (masters3d) [inline exercises/space-age/SpaceAgeTest.swift]: Please remove this print statement.

**Comment by masters3d** [comment]
Good idea. 
Do you mind doing a PR for the swift3 branch as well?

https://github.com/exercism/xswift/blob/swift3/exercises/space-age/SpaceAgeTest.swift

**Comment by srabuini** [comment]
Fixed! That print was the motivation of this PR.

I'll do a PR for the swift3 branch too.

**Comment by masters3d** [comment]
Thank you!

**Pull request merged**